projects/dotcms-field-elements/src/components/dot-key-value/key-value-form/key-value-form.e2e.ts

```diff
@@ -212,6 +198,10 @@ describe('key-value-form', () => {
                 expect(spyAddEvent).not.toHaveReceivedEvent();
             });
         });
+
+        xdescribe('lostFocus', () => {
```

Comment:
Can't test blur, the trick @freddyucv doesn't work.

Reply:
remove it then

Reply:
Need to find a way to test it